test/cli_test.go

```diff
@@ -105,10 +105,6 @@ func (s *CLITestSuite) runIntegrationTest(tt CLITest) {
 		tt.name = tt.args
 	}
 
-	if strings.HasPrefix(tt.name, "error") {
```

Comment:
This looked flaky. CI is passing so I assume all `exitcode=1` are caught

Reply:
Yep, I made sure to add `exitCode: 1` to all of the tests this line applied to.